### ISSUE TITLE
Financial Services Support Specialist applies for Merchant Tax role

### ISSUE THREAD

**Issue by smglinski** [smglinski/Glinski#1]
**TL;DR:**
I am excited to apply for this Support Specialist, Merchant Tax role because I believe I can have my best impact on Shopify's mission by helping merchants understand the complicated and important subject of tax. I've practised making information simple and accessible throughout my 4 years in support, and I hope this experience provides a valuable perspective as we grow the team. 

**Background Information:**
I have worked as a Financial Services Support Specialist on the Billing and Shopify Payments team for over 2 years. In that time I have dealt with a variety of money-related problems, which involves working with teams to draft communications and fix technical issues. This experience has taught me to naturally make decisions that are fair for both merchants and Shopify with little personal bias, and my writing shows that I can do this with enduring empathy.

**Overview and Examples:**
- "Wrong" Shopify Payments rates - https://shopify.zendesk.com/agent/tickets/15993893:
 It was awkward when a former Shopifolk called in to complain about having been quoted the wrong Shopify Payments fee rates for their Alumni plan. Our communication on their plan change had been poor, but it was not enough to warrant a refund. I addressed each single complaint with a clear explanation, and I believe they were satisfied. I highlight all of my replies on this ticket for you to review. 

- Responding to "I hate working with all of you" - https://shopify.zendesk.com/agent/tickets/16973159:
I am highlighting my reply from April 17 at 16:49 which attempts to calmly explain to the merchant how we will send him his payouts which had failed. I am proud of how I kindly corrected their assertions, gave them the most important information first, and chose to not include the information they demanded because I believed it would not help (and explained that to them.)

**Solution:**
I enjoy my current role in Financial Services Support because with my own eyes I can see the contribution we make to Shopify's mission by making financial services better for everyone. I've worked to support this with small contributions like presentations to Guru teams and fixing the Shopify Payments Japan setup form, and I know I can do much more. The prospect of becoming an expert in this new area of financial services support so that I can help others more easily understand it really excites me. If you think I would be a good fit for the role, please contact me so we can talk about it. You can reach me on Slack @steven or by using the contact details provided.

**Comment by smglinski** [comment]
Before we ship this I'll share a personal story of failure. I'm always looking for ways to make money so I can do anything related to auto racing. I know a lot about cars, so I've tried to purchase cars to resell them and make a profit. The first (recent) one was a 2010 Nissan Altima. I didn't do enough research, and instead of a long list of mechanical issues, I'll just let you know I lost about $2000 on the whole endeavour. 

I looked and that loss and told myself I just had a very expensive lesson in used car sales. I'll use that wisdom to avoid future losses. 

But...a few months later I tried again on a beautiful 2007 Lexus RX 350 when I bought it for what seemed like a steal. A few mechanical issues were beyond my repair expertise and I lost $1500. 

I believe the most valuable lesson I've taken from this is the need for patience. I expended more personal energy than I stood to gain by trying to take a shortcut, and I lost anyways. I may even buy another neglected car and do the same thing again in the future because part of it is fun, but it's not a feasible get-rich-quick scheme. I'll put a certain amount of my energy into my career, and that's how I'll earn money. The rest goes towards being happy and being a good person. If an opportunity comes to me, I'll welcome it, but I don't need to put myself in a precarious position to make a buck.

**Comment by smglinski** [comment]
Stopped adding labels to this issue as we realized GitHub leaves a record of anything. If this were a job application, the reader would be able to see the applicant's mindset as they decided which personal traits to highlight. Good thing it's just a GitHub issue. 

Ship it! 🚀
